src/components/General/LeftSidebar/LeftSidebar.tsx

```diff
@@ -51,15 +51,27 @@ const LeftSidebar = () => {
       }}
     >
       {buttons.map((button: any) => (
-        <LeftSidebarIconButton
-          key={button.id}
-          iconEl={renderIcon(button.id)}
-          isDotOn={button.isDotOn}
-          isSelected={button.isSelected}
-          isDisabled={button.isDisabled}
-          id={button.id}
-          path={button.path}
-        />
+        <Tooltip
+          content={
+            <XStack alignItems="center" space="$1">
+              <Text size={15} weight="semibold">
+                {button.path.slice(1)}
```

Comment:
Can we have a function that maps the path to more human-friendly text? 
For example instead of `device-health-chek` it will be better to see `Device health check`